src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.Print.cs

```diff
@@ -25,16 +28,36 @@ internal static unsafe string DerStringToManagedString(byte[] derString)
                 return null;
             }
 
+            using (asn1String)
+            {
+                return Asn1StringToManagedString(
+                    asn1String,
+                    (bio, str, flags) => Asn1StringPrintEx(bio, str, flags));
```

Comment:
So have I been doing it "worse"?

https://github.com/dotnet/corefx/blob/master/src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CollectionBackedStoreProvider.cs#L125